### PR TITLE
eslint added

### PR DESCRIPTION
**Pull request by arhamimran221**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      "no-param-reassign": ["error", { "props": true, "ignorePropertyModificationsFor": ["state"] }],
+      "import/prefer-default-export": "off",
+      "react/prop-types": "off",
+      "react/function-component-definition" : "off",
+      "jsx-a11y/label-has-associated-control" : "off",
+      "prefer-destructuring" : "off"
+    }
+    
+  }
+
```
Comment (Taimoormk) [inline .eslintrc.json:52]: EOF - end of file, there should be an empty line in the end of file

File: .next/build-manifest.json [hunk 2]
```diff
@@ -15,16 +13,7 @@
...
-    ],
-    "/_error": [
-      "static/chunks/webpack.js",
-      "static/chunks/main.js",
-      "static/chunks/pages/_error.js"
-    ]
+    "/_app": []
   },
   "ampFirstPages": []
 }
```
Comment (Taimoormk) [inline .next/build-manifest.json:19]: .next folder should not be committed, it should be in the gitignore file.

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "semi": false,
+    "singleQuote": true,
+    "tabWidth": 2,
+    "printWidth": 80
+  }
+
```
Comment (Taimoormk) [inline .prettierrc:7]: EOF - end of file, there should be an empty line in the end of file

File: src/app/StoreProvider.jsx [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 import { Provider } from 'react-redux'
-import { store } from '../Redux/Store.js';
+import { store } from '../Redux/Store';
 
 export default function StoreProvider({ children }) {
 
   return <Provider store={store}>{children}</Provider>
-}
\ No newline at end of file
+}
```
Comment (Taimoormk) [inline src/app/StoreProvider.jsx:9]: EOF - end of file, there should be an empty line in the end of file

File: .eslintrc.json (hunk 1 above)
Comment (arhamimran221) [reply .eslintrc.json:52]: Ok sir

File: .next/build-manifest.json (hunk 2 above)
Comment (arhamimran221) [reply .next/build-manifest.json:19]: Ok sir

File: src/app/StoreProvider.jsx (hunk 4 above)
Comment (arhamimran221) [reply src/app/StoreProvider.jsx:9]: Ok sir

**Pull request merged**